packages/solid-start-client/src/StartClient.tsx

```diff
@@ -15,7 +18,21 @@ export function StartClient(props: { router: AnyRouter }) {
   return (
     <Await
       promise={hydrationPromise}
-      children={() => <RouterProvider router={props.router} />}
+      children={() => (
+        <Dummy>
+          <Dummy>
+            <RouterProvider
+              router={props.router}
+              InnerWrap={(props) => (
```

Comment:
do we need this innerWrap?

Reply:
Yeah it's for matching the component tree structure between server and client